### PR TITLE
Mustafa_Aisal was doing struggling already and Victor came in to destroy it!!

### PR DESCRIPTION
**Pull request by MustafaOk**

### CODE REVIEW

File: src/test/java/Mustafa_Aisal.java [hunk 1]
```diff
@@ -2,6 +2,12 @@
     //Mustafa-cucumber-tomatoes-pepper-blackpepper
     //mustafa-my dinner is ready..
     public static void main(String[] args) {
+        System.out.println("Conflict's will never end!!!");
```
Comment (IgorTirsina) [inline src/test/java/Mustafa_Aisal.java:5]: lets add a loop for "conflict will never end"

Comment (MustafaOk) [reply src/test/java/Mustafa_Aisal.java:5]: in progress..

**Review by IgorTirsina — COMMENTED** [review]
Lets add a loop so we have a clean code

File: src/test/java/Mustafa_Aisal.java (hunk 1 above)
Comment (aliakpa) [reply src/test/java/Mustafa_Aisal.java:5]: recursice conflict. we just learnt how to handle one conflict, how are we gonna handle infinite loop of conflict :)

**Review by VictorIachim — CHANGES_REQUESTED** [review]
Please add that in a loop

**Review by IgorTirsina — APPROVED** [review]

**Review by IgorTirsina — COMMENTED** [review]
the loop will be an error, so lets use i<5

**Review by IgorTirsina — APPROVED** [review]
Lets add some changes?

**Review by aliakpa — APPROVED** [review]

**Review by IgorTirsina — CHANGES_REQUESTED** [review]
add i<5 please

**Comment by aliakpa** [comment]
flawless

File: src/test/java/Mustafa_Aisal.java [hunk 2]
```diff
@@ -3,6 +3,10 @@
     //mustafa-my dinner is ready..
     public static void main(String[] args) {
 
+        for (int i = 0; i > 0 ; i++) {
```
Comment (IgorTirsina) [inline src/test/java/Mustafa_Aisal.java:6]: for (int  i = 0; i < 5; i++);

**Review by MustafaOk — COMMENTED** [review]
Sasfagf

**Review by VictorIachim — APPROVED** [review]
Goog job

**Review by VictorIachim — APPROVED** [review]
no bueno

**Review by VictorIachim — APPROVED** [review]
balbala

**Review by VictorIachim — APPROVED** [review]
asdasd

**Review by IgorTirsina — APPROVED** [review]
i aprrove but dont delete the branches

**Pull request merged**